src/controls/ValueEditor.tsx

```diff
@@ -271,8 +273,8 @@ const ValueEditor: React.FC<ValueEditorProps> = (props) => {
     case 'textarea':
       return renderTextArea({ ...props, onTextAreaChange, _value, inputDisabled });
     case 'custom':
-      if (customRenderer) return customRenderer({ ...props, onChange: (val) => onCustomRendererChange(val, handleOnChange)});
-      return <></>
+      if (customRenderer && getSelectionKey && field) return customRenderer({ ...props, onChange: (val) => onCustomRendererChange(val, handleOnChange,  getSelectionKey(field))});
```

Comment:
const key = getSelectionKey?.(field) || "id";
if (customRenderer ) return customRenderer({ ...props, onChange: (val) => onCustomRendererChange(val, handleOnChange, key)});

try like this, if getselectionkey undefined means take from id key.